html/classhelper.js

```diff
@@ -303,7 +312,7 @@ class ClassHelper extends HTMLElement {
             throw new Error(message);
         }
 
-        ClassHelper.translations = translations;
+        ClassHelper.translations = json;
```

Comment:
Okay, this is an artifact from another local branch that I was trying to solve, translations stopped working

This solves the problem where I was not attaching the API response to the member field